operator/templates/stateful-set.yaml

```diff
@@ -46,7 +46,7 @@ spec:
           imagePullPolicy: Always
           # Dockerfile based on:
           # https://github.com/docker-library/cassandra/blob/master/3.11/Dockerfile
-          image: mpereira/cassandra:3.11.4
+          image: viivek46/cassandra:3.11.4
```

Comment:
hm, can we add tooling to use non-personal images? and can we add additional versioning so that we don't overwrite each other's images?

Reply:
We don't need to use the Cassandra image, right?

Here's what I propose in terms of KUDO Cassandra depending on the cassandra_exporter repository:
1. Add cassandra_exporter it as a git submodule under `vendor/cassandra_exporter`
2. Create a symlink from `images/Dockerfile.prometheus-exporter` to `vendor/cassandra_exporter/docker/Dockerfile`
3. Build/push `mesosphere/cassandra_exporter` Docker image based on `images/Dockerfile.prometheus-exporter` (can be personal for now until I finish implementing Docker image build support) and use that for the prometheus exporter container.
4. Revert the changes in the Cassandra node Dockerfile and pod.

Reply:
This sounds kind of involved - why don't we make this a separate ticket for now so that we don't block the metrics work. If we're okay with personal images for now, let's continue using them until the Docker image build support is in place.

Reply:
Yeah, step 3 above calls out that the image can be personal.

Reply:
Please revert line 49 for now @viivek46.

Reply:
we have to modify exporter docker file now, as cassandra_exporter.jar file was not accessible by non root user, i think we should skip step 1,2 from Murilo's suggestion and copy paste run.sh and docker file from crito's repo and modify them in our repo.

WDYT @mpereira @samvantran